<?xml version="1.0" encoding="UTF-8"?>
<root BTCPP_format="4">
  <!-- Description of Node Models (used by Groot) -->
  <TreeNodesModel>
    <Action ID="SetIO" editable="true">
        <input_port name="service_name" default="/io_and_status_controller/set_io"/>
        <input_port name="function"/>
        <input_port name="pin"/>
        <input_port name="state"/>
    </Action>
    <Action ID="GetRobotMode" editable="true">
      <input_port name="service_name" default="/ur_dashboard_client/robot_mode"/>
      <output_port name="robot_mode"/>
    </Action>
    <Action ID="SetRobotModeRunning" editable="true">
      <input_port name="action_name" default="/ur_dashboard_client/set_mode"/>
    </Action>
  </TreeNodesModel>
</root>
